jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -39,6 +41,34 @@ CircleCI automatically creates a new branch and commits a basic configuration fi
 You can choose a sample `.circleci/config.yml` to best fit your project from a variety of templates (for example, Node.js, Python, iOS apps). You can make changes to the file in CircleCI before saving. The config is committed on a new branch.
 
 The first two options (**Fastest** and **Faster**) automatically trigger a pipeline once you create the project.
+--
+
+[.tab.signup.GitLab_Self-Managed]
+--
+You need:
+
+* Your GitLab self-managed instance URL, e.g. `https://example.circleci.com``.
++
+Your self-managed instance must already contain at least one project. The authorization attempt will be unsuccessful if your instance does not have any projects.
+
+* A link:https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html[personal access token]. This token must have the `api` scope.  
+
+* Your instance's SSH key fingerprint. You can copy this from your instance's `known_hosts` file, which is located by default in the `~/.ssh/` directory.
+
+To create a new project:
+
+. Enter your self-managed instance URL. The field will automatically append `/api/v4`, so the URL that CircleCI uses to connect to your account looks something like `https://example.circleci.com/api/v4`.
+
+. Enter your personal access token. Click **Connect**.
+
+. Copy and paste the instance SSH key fingerprint into **Known hosts file**. Click **Connect**.
+
+. Once your account is successfully authorized, select the repository for your CircleCI project, and enter a project name. 
+
+. Finally, click **Create Project**. You will then be redirected to the CircleCI projects dashboard. 
+
+The express CircleCI configuration setup is not currently available for GitLab Self-managed projects. You will need to add a `.circleci/config.yml` file in your repository if it has not yet been set up with a CircleCI config. If the repository you selected already contains a `.circleci/config.yml`, you will need to save a commit in the repo to see your pipeline running in the dashboard.
```

Comment:
```suggestion
The express CircleCI configuration setup is not currently available for GitLab Self-managed projects. You will need to add a `.circleci/config.yml` file in your repository if it has not yet been set up. If the repository you selected already contains a `.circleci/config.yml`, you will need to save a commit in the repo to see your pipeline on the dashboard.
```

Reply:
Just a terminology suggestion - pipelines are 'triggered" not "run", only jobs and workflows "run" ... this is buried in the style guide somewhere